src/Joonasw.AspNetCore.SecurityHeaders/Hsts/HstsOptions.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System;
 
-namespace Joonasw.AspNetCore.SecurityHeaders
+namespace Joonasw.AspNetCore.SecurityHeaders.Hsts
```

Comment:
Same thing here, please put the namespace + file back to avoid a breaking change.